 
#Goal
int32 x_pix # top left corner pixel x 
int32 y_pix # top left corner pixel y
int32 x_width # crop width
int32 y_width # crop width

---
#Result
sensor_msgs/Image result_img

---
#Feedback
sensor_msgs/Image feedback_img